notes

```diff
@@ -0,0 +1,12 @@
+make docker images use variables
```

Comment:
Agreed using variables for the docker image tags would be a cool next step.

Apologies if these are things you already know, but breadcrumbs I've found:
- In order to get CircleCI to use a `.env` or other environment file you have to read the environment variables into the bash context in a separate step. (like, have to write a bash script to set them)
- The only environment variables it will mask are those set within CircleCI itself, in either the project (just that one repo), or the context (a set of projects). 
- You've already found the parameters thing, which I still don't fully understand but which seems handy